test/jdk/sun/util/calendar/zi/TestZoneInfo310.java

```diff
@@ -280,5 +282,19 @@ private static ZoneInfoOld toZoneInfoOld(TimeZone tz) throws Exception {
                                willGMTOffsetChange.getBoolean(tz));
     }
 
+    private static void diff (String[] o, String[] n) {
+        var l1 = Arrays.asList(o);
+        var l2 = Arrays.asList(n);
 
+        if (l1.equals(l2)) {
+            System.out.println("no diff");
```

Comment:
Is this needed? IIUC `diff` is only called if the array contents are not equal.

Reply:
Since `diff` is a generic debug function, equality check needed to be in the function itself. Anyway, I wrote this code for purely debug purpose, so can be removed altogether.